framework/utils/codeEditorUtils.ts

```diff
@@ -1,52 +1,12 @@
 import jsyaml from 'js-yaml';
 
-export function yamlToJson(yamlString: string) {
-  const value: object = jsyaml.load(yamlString) as object;
-  if (!value) {
-    return '{}';
-  }
-  if (typeof value !== 'object') {
-    throw new Error('yaml is not in object format');
-  }
-  return JSON.stringify(value, null, 2);
-}
-
 export function jsonToYaml(jsonString: string) {
   if (jsonString.trim() === '') {
-    return '---\n';
+    return '';
   }
   const value = JSON.parse(jsonString) as object;
   if (Object.entries(value).length === 0) {
-    return '---\n';
+    return '';
   }
   return jsyaml.dump(value);
 }
-
```

Comment:
Those functions might be nice to keep around...since we deal with JSON/YAML conversion all the time.